traffic_ops/testing/api/v4/servercapabilities_test.go

```diff
@@ -83,6 +96,29 @@ func GetTestServerCapabilities(t *testing.T) {
 	}
 }
 
+func UpdateTestServerCapabilitiesWithHeaders(t *testing.T, header http.Header) {
+	resp, _, err := TOSession.GetServerCapabilitiesWithHdr(header)
+	if err != nil {
+		t.Fatalf("Expected no error, but got %v", err.Error())
+	}
+	if len(resp) == 0 {
+		t.Fatal("no server capability in response, quitting")
+	}
+	originalName := resp[0].Name
+	newSCName := "sc-test"
+	resp[0].Name = newSCName
+
+	if (resp) != nil {
```

Comment:
if `len(resp) != 0`, can `resp` ever be `nil`?

Reply:
nope. i'll remove the redundant check.